zivid_description/urdf/macros/zivid_camera.xacro

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+    <xacro:macro name="zivid_camera" params="prefix">
+        <!-- Zivid Properties -->
+        <xacro:property name="M_PI" value="3.1415926535897931" />
+
+        <!-- Zivid Base Link -->
+        <link name="${prefix}base_link">
+            <!-- Visuals -->
+            <visual>
+                <origin xyz="-0.0030 -0.0758 0.0445" rpy="${0.5*M_PI} 0 ${0.5*M_PI}"/>
+                <material name="blue">
+                    <color rgba="0.25 0.25 0.25 1"/>
```

Comment:
It's slightly confusing the material name is "blue", while the actual provided rgba value is grey.

Reply:
Agree, that's a left-over from copy-pasting ;). I'll update this

Reply:
Nice, looks good now 👍